packages/create-sitecore-jss/src/templates/vue/src/lib/dictionary-service-factory.js

```diff
@@ -1,16 +1,16 @@
 import {
   DictionaryService,
-<% if (fetchWith.toUpperCase() === 'GRAPHQL') {-%>
+<% if (fetchWith.toUpperCase() === 'GRAPHQL') { -%>
   GraphQLDictionaryService
-  <% } else if (fetchWith.toUpperCase() === 'REST') {-%>
+  <% } else if (fetchWith.toUpperCase() === 'REST') { -%>
       RestDictionaryService
       <% } -%>
 } from '@sitecore-jss/sitecore-jss-angular';
 import config from '../temp/config';
 
 export class DictionaryServiceFactory {
   create() {
-  <% if (fetchWith.toUpperCase() === 'GRAPHQL') {-%>
+  <% if (fetchWith.toUpperCase() === 'GRAPHQL') { -%>
```

Comment:
These ejs conditionals are still being indented, can we keep them lined up?

Reply:
i guess i need to change the whole vs code editor setting. 
As soon as I save the file these conditional get messed up.